packages/prisma/schema.prisma

```diff
@@ -260,6 +261,8 @@ model Booking {
   rejectionReason     String?
   dynamicEventSlugRef String?
   dynamicGroupSlugRef String?
+  rescheduled         Boolean?
+  fromReschedule      String?
```

Comment:
fromReschedule can be easily removed as it only saves previous booking Uid, but after reschedule previous bookings gets deleted. I want to hear your thoughts on this